### ISSUE TITLE
Jabber sessions are gone [regression]

### ISSUE THREAD

**Issue by Sbte** [emesene/emesene#918]
f0680a73770e2f885e31c3ce2af42619882ef6e7 is the first bad commit
commit f0680a73770e2f885e31c3ce2af42619882ef6e7
Author: 4ndreaSt4gi stagi.andrea@gmail.com
Date:   Thu Nov 3 23:34:00 2011 +0100

```
e3.jabber: moved mail client in Session, added Facebook mail counter
```

:040000 040000 c2b94977bbc3c29d76423e424aaf6de30ae6d930 c166b5701eddbb41b8e51b42f4900cbf4ce69191 M  emesene

@4ndreaSt4gi^

**Comment by astagi** [comment]
?? all works fine here.. there was something missing, but I fixed it... what's your problem?

**Comment by Sbte** [comment]
That the Facebook and gtalk sessions are gone from the combobox on the login screen.

**Comment by astagi** [comment]
Uhm really strange ... In any case I'll have a look tonight and try to solve it...

Thank you

**Comment by Sbte** [comment]
Traceback (most recent call last):
  File "/home/sven/emesene-dev/emesene/emesene", line 32, in <module>
    import emesene
  File "/home/sven/emesene-dev/emesene/emesene.py", line 92, in <module>
    from e3 import jabber
  File "/home/sven/emesene-dev/emesene/e3/jabber/**init**.py", line 2, in <module>
    from Session import Session
  File "/home/sven/emesene-dev/emesene/e3/jabber/Session.py", line 4, in <module>
    from MailClients import *
  File "/home/sven/emesene-dev/emesene/e3/jabber/MailClients.py", line 155, in <module>
    from pyfacebook import Facebook
  File "/home/sven/emesene-dev/emesene/e3/jabber/pyfacebook/**init**.py", line 61, in <module>
    from django.conf import settings
ImportError: No module named django.conf

**Comment by astagi** [comment]
Oh... it's a pyfacebook issue... it's deprecated, they didn't pay attention on who uses their library without having Django.. I have Django installed, so this bug passed silently... I'll fix in a moment ^^

**Comment by astagi** [comment]
Forgot to specify in commit. Anyway closed by dd0e2bde3a167b884736a5ae9462a2d51c2b52d6
